devel/kubetail/Portfile

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+# Portfile
+PortSystem          1.0
+PortGroup           github 1.0
+
+# ── Basic metadata ─────────────────────────────────────────────────────────────
+name                kubetail
+version             0.8.2
```

Comment:
github.setup line should be at the top and the "name" and "version" will be set from there.